src/snowflake/snowpark/udtf.py

```diff
@@ -421,6 +421,99 @@ class UDTFRegistration:
             -----------------------------
             <BLANKLINE>
 
+    [Preview Feature] Syntax for declaring vectorized UDTF process method is similar to above. Defining
+    ``__init__`` and ``end_partition`` methods are optional. ``process`` method only accepts one argument
+    which is the pandas dataframe object, and outputs the same number of rows as is in the given input. Both
+    ``__init__`` and ``end_partition`` do not take any additional
+    arguments.
+
+    Example 15
+        vUDTF process method without end_partition
```

Comment:
vUDTF -> Vectorized UDTF

Please change this also in every instance that appears below this.